controllers/model/grafanaDeployment.go

```diff
@@ -155,8 +155,8 @@ func getNodeSelectors(cr *v1alpha1.Grafana) map[string]string {
 
 func getTerminationGracePeriod(cr *v1alpha1.Grafana) *int64 {
```

Comment:
If we change that value to a pointer in the CR, do we need that function at all? The deployment also expects a pointer, so we could just pass it through from the CR to the deployment. This is assuming we want to use the Kubernetes default if the users isn't setting it.

Reply:
I'll test it out.

Reply:
It works without that function now so I made the change.